spec/fixtures/responses/queries/freaks/freaks_pagination.json

```diff
@@ -4,91 +4,13 @@
     "data": {
       "freaks": {
         "pageInfo": {
-          "endCursor": "MQ",
-          "startCursor": "MQ",
+          "endCursor": null,
+          "startCursor": null,
```

Comment:
This is wrong. Why delete the expected response? This file & it's spec should suffer no changes.